heroku-20-cnb-build/Dockerfile

```diff
@@ -9,8 +9,10 @@ RUN mkdir /app && \
 
 ENV CNB_USER_ID=1000
 ENV CNB_GROUP_ID=1000
-
 ENV CNB_STACK_ID "heroku-20"
+ENV CNB_TARGET_OS="linux"
```

Comment:
For consistency, would it make sense to use `$TARGETOS` etc for these?

Or are they only available when using buildx?

That said, is there a reason why heroku-20/22 aren't using buildx too?
https://github.com/heroku/base-images/blob/d1ab0cb6a28e67c5a42f95e86c661dee1523768c/bin/build.sh#L54-L69

...I would guess to reduce number of changes to existing stacks?  But longer term should we consolidate and use buildx for the older stacks too (albeit in single arch mode)?

Reply:
> Or are they only available when using buildx?

Yes, that's my understanding. Read that here: https://www.docker.com/blog/faster-multi-platform-builds-dockerfile-cross-compilation-guide/.

> That said, is there a reason why heroku-20/22 aren't using buildx too? I would guess to reduce number of changes to existing stacks?